tests/index.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+require( './config' )();
+const { runTests } = require( './_helpers/test-runner' );
+
+const tests = [
```

Comment:
Ok, so it will be good to have the ultimate test for our runner. Now we have only two tests here (since we have two testable workflows only). But both workflows can have different internal flows.

So `setup-workflows.yml` may push directly to master or create a PR. Same for `update-deps.yml`, it can have `targetBranch` changed via config or can be run on repo withut `package.json` file (btw. it is missing in this test now).

If we are able to create reasonable working tests covering all the above, we are good to go. WDYT?